iamproperty/iamproperty.github.io
# Ds 10/get started

## Summary of Changes:
1. Update readme file and populate the Get started page
2. Remove highlight js and use a CDN version instead (Just easier and remove a dependency)
3. Tests packaging up library and use in external apps
4. Create a pull request task

## Rationale
Add more information to the get started page

## How to Test
Open up the preview site (see the below netlify comment below) and navigate to /get-started

## Ticket(s)
DS-10

## Checklist

## Notes
-
